src/FluidXPlate/FluidXPlate.php

```diff
@@ -10,7 +10,8 @@
 
 final class FluidXPlate
 {
-    public const FLUIDX_BARCODE_REGEX = /* @lang RegExp */ '/[A-Z]{2}(\d){8}/';
+    public const FLUIDX_BARCODE_REGEX = /* @lang RegExp */ '/' . self::FLUIDX_BARCODE_REGEX_WITHOUT_DELIMITER . '/';
+    public const FLUIDX_BARCODE_REGEX_WITHOUT_DELIMITER = '[A-Z]{2}(\d){8}';
```

Comment:
@spawnia I'm not really happy with this part, but I don't want to repeat myself either. so this is a valid, but ugly solution. What do you think?

Reply:
It is fine if you want to avoid duplication. If you want to go the other way, I would only use literal regexes to improve IDE support (makes `@lang RegExp` work nicely, allows ad-hoc testing via PhpStorm).